incident_archive/ir_version_1_0.md

```diff
@@ -0,0 +1,69 @@
+# Archive Notice
+**IMPORTANT:** This page is retained for historical purposes only.
+
+For the latest version of the CCADB Incident Reports, see https://www.ccadb.org/cas/incident-report.
+
+----
```

Comment:
```suggestion
Version 1.0
----
```

We should clearly identify the Version # of this archived document (beyond just the filename). If not here, consider as a page Title (like we do with [CRP Policy archive](https://www.chromium.org/Home/chromium-security/root-ca-policy/policy-archive/version-1-2/))